gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -315,6 +315,7 @@ main (int argc, char *argv[])
   /* Upstream has "existing user" mode for new user accounts. In Endless, we
      skip straight to the welcome tutorial instead. */
   if (get_mode () == GIS_DRIVER_MODE_EXISTING_USER) {
+    setlocale (LC_ALL, "");
```

Comment:
I won't insist on this point, given that we're really close to the end game for 2.6.0.
But my concern is that we later do something else that requires us to initialize GTK and we have to do that anyway (or we're bitten by it because we don't realize we should have initialized it...)